app/views/admin/categories/edit.html.erb

```diff
@@ -0,0 +1,37 @@
+<%= render "admin/sidebar_admin" %>
+<div class = "col-md-9">
+  <%= form_with(model: [:admin, @category], local: true) do |c|%>
```

Comment:
Tìm hiểu thêm form for, form tag

Reply:
Em tìm hiểu thấy, form with nó có chức năng giống cả 2 cái form for form tag. Nên em thấy chỗ này có thể dùng form with.